sim/shaman/items.go

```diff
@@ -163,81 +78,6 @@ var ItemSetSkyshatterHarness = core.NewItemSet(core.ItemSet{
 })
 
 func init() {
-	core.NewItemEffect(19344, func(agent core.Agent) {
```

Comment:
I thought we were keeping item/set effects? Otherwise we should not allow TBC items to be included in UI.